### PR TITLE
Update to beta note on Session Replay SDK Docs

### PR DESCRIPTION
**Pull request by jas-kas**
To encourage users to open GitHub issues for their feedback, and to list the supported SDKs on the Session Replay SDK docs.

### CODE REVIEW

File: src/includes/beta-note-session-replay.mdx [hunk 1]
```diff
@@ -1,5 +1,16 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please open a [GitHub issue](https://github.com/getsentry/sentry/issues/new/choose). Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```
Comment (billyvg) [inline src/includes/beta-note-session-replay.mdx:3]: Should we ask them to also include a link to the replay and/or the page where they are having issues?

File: src/includes/beta-note-session-replay.mdx [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+**Currently, this feature is supported for only browser JavaScript, including these frameworks:**
+
+- Angular
```
Comment (JoshFerge) [inline src/includes/beta-note-session-replay.mdx:7]: do we need to list these frameworks explicitly? as we'll work on other frameworks, there's just not a specific sentry sdk for them.

File: src/includes/beta-note-session-replay.mdx (hunk 1 above)
Comment (jas-kas) [reply src/includes/beta-note-session-replay.mdx:3]: that's a good idea! also realized this beta note is on SDK docs, so going to change the repo link to `sentry-javascript`

File: src/includes/beta-note-session-replay.mdx (hunk 2 above)
Comment (jas-kas) [reply src/includes/beta-note-session-replay.mdx:7]: We have this list on our SDK docs on our product docs - https://docs.sentry.io/product/session-replay/. Considering the volume of inquiries around if we formally support Electron, iOS, React Native, etc. I thought it might be helpful to repeat it here since these docs are viewed often

Maybe we could shorten this to something like - "Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs." instead of individually listing? What do you think?

File: src/includes/beta-note-session-replay.mdx (hunk 1 above)
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:3]:
```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or mention the Sentry page you were viewing. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```

Comment (jas-kas) [reply src/includes/beta-note-session-replay.mdx:3]: cc/ @billyvg

Comment (billyvg) [reply src/includes/beta-note-session-replay.mdx:3]:
```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or if possible a publicly accessible URL to the page you are attempting to record a replay of. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```

**Review by billyvg — APPROVED** [review]

File: src/includes/beta-note-session-replay.mdx [hunk 3]
```diff
@@ -1,5 +1,16 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or if possible a publicly accessible URL to the page you are attempting to record a replay of. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+
```
Comment (lizokm) [inline src/includes/beta-note-session-replay.mdx:4]:
```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).

```

**Review by lizokm — APPROVED** [review]
Just made it sound a bit more casual :)

File: src/includes/beta-note-session-replay.mdx (hunk 2 above)
Comment (JoshFerge) [reply src/includes/beta-note-session-replay.mdx:7]: sounds good to me :) may be worth mentioning that it _doesn't_ currently support "Electron, iOS, React Native"

File: src/includes/beta-note-session-replay.mdx [hunk 4]
```diff
@@ -1,5 +1,15 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:4]:
```suggestion
Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs.
```

File: src/includes/beta-note-session-replay.mdx [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+
+- Angular
+- Ember
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:7]:
```suggestion

```

File: src/includes/beta-note-session-replay.mdx [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+- Gatsby
+- Next.js
+- React
+- Remix
+- Svelte
+- Vue
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:13]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 7]
```diff
@@ -1,5 +1,14 @@
...
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
+
+- Angular
+- Ember
+- Gatsby
+- Next.js
+- React
+- Remix
+- Svelte
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:12]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 8]
```diff
@@ -1,5 +1,13 @@
...
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
+
+- Angular
+- Ember
+- Gatsby
+- Next.js
+- React
+- Remix
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:11]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 9]
```diff
@@ -1,5 +1,12 @@
...
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
+
+- Angular
+- Ember
+- Gatsby
+- Next.js
+- React
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:10]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 10]
```diff
@@ -1,5 +1,11 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
+
+- Angular
+- Ember
+- Gatsby
+- Next.js
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:9]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 11]
```diff
@@ -1,5 +1,9 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+**Currently, this feature is only supported for browser JavaScript, including these frameworks:**
+
+- Angular
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:6]:
```suggestion
```

File: src/includes/beta-note-session-replay.mdx [hunk 12]
```diff
@@ -1,5 +1,9 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+
+Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs.
+
+- Gatsby
```
Comment (jas-kas) [inline src/includes/beta-note-session-replay.mdx:7]:
```suggestion
```
